src/main/java/com/mx/iwaconsolti/calculator/persistence/model/Multiply.java

```diff
@@ -0,0 +1,20 @@
+package com.mx.iwaconsolti.calculator.persistence.model;
+
+import org.springframework.stereotype.Component;
+
+@Component
+public class Multiply {
+    private double num1, num2;
```

Comment:
This is not recommended to do using beans. 

The beans should try to be stateless, so they dont store any value, since other person can change it value at any time, since we only have one instance of class

Reply:
I already modified the class to prevent it from storing data